modules/ROOT/pages/deploy-to-cloudhub.adoc

```diff
@@ -99,9 +99,8 @@ If not set, by default this value is set to +https://anypoint.mulesoft.com+. | N
 | `muleVersion` | The Mule runtime engine version to run in your CloudHub instance.  +
 Ensure that this value is equal to or higher than the earliest required Mule version of your application. +
 Example value: `4.3.0` +
-
-Starting with Mule 4.5, deployments to CloudHub require Major.Minor version, which deploys the latest version of Mule runtime. 
-Example value: `4.5`| Yes
+Starting with Mule 4.5, deployments to CloudHub accepts the Mule version exact name or the semantic versioning, which deploy the latest version of Mule runtime. +
+Example values: `4.6`, `4.6-java8` , `4.6e-java17` | Yes
```

Comment:
We also need 2 new rows for 1) javaVersion 2) releaseChannel fields -- which is missing at this point